ext/opentelemetry-ext-pymongo/src/opentelemetry/ext/pymongo/__init__.py

```diff
@@ -115,7 +122,12 @@ def failed(self, event: monitoring.CommandFailedEvent):
         if span is None:
             return
         span.set_attribute("db.mongo.duration_micros", event.duration_micros)
-        span.set_status(Status(StatusCanonicalCode.UNKNOWN, event.failure))
+        span.set_status(
+            Status(
+                StatusCanonicalCode.UNKNOWN,
+                json.loads(bson.json_util.dumps(event.failure)),
```

Comment:
will this stringify all expected types? One thing I see is that event.failure is a more structured document. the Status description must be a string, or it will be discarded.

Instead of json.loads, should this be a str() cast?